Support for Azure OpenAI + Ollama + unit tests for the embeders and LLMs (#140)
neo4j/neo4j-graphrag-python: docs/source/user_guide_rag.rst

Using Another LLM Model
========================

> If OpenAI can not be used directly, there are a few available alternatives:

Reviewer: ```suggestion
If OpenAI cannot be used directly, there are a few available alternatives:
```
